src/App.js

```diff
@@ -1,14 +1,57 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import axios from 'axios';
 import "./App.css";
+import { InputDate } from './components/InputDate';
+import { MediaUrl } from './components/MediaUrl';
+import { Explanation } from './components/Explanation';
+import { PhotoDate } from './components/PhotoDate';
+import { Title } from './components/Title';
+
+const apiKey = 'vSbHMwWWl96BgKXN2Qxqm8JLJ17HoWsqlC0KAUMK';
+const photoOfTheDayApi = `https://api.nasa.gov/planetary/apod?api_key=${apiKey}`;
 
 function App() {
+  const [mediaUrl, setImage] = useState('');
+  const [mediaType, setMediaType] = useState('');
+  const [title, setTitle] = useState('');
+  const [explanation, setExplanation] = useState('');
+  const [date, setDate] = useState('');
```

Comment:
These lines can be reduced by saving the entire photo object in one slice of state e.g

```
const [photo, setPhoto] = useState(null);